src/helpers/file.helpers.ts

```diff
@@ -19,6 +19,27 @@ const isConfigured = (config: Config) => {
     isAuthConfigured = isAuthUsernameConfigured && isAuthPasswordConfigured;
   }
   return isProjectKeyConfigured && isSonarURLConfigured && isAuthConfigured;
+}
+
+export function isPullRequestOrBranchConfigured(config: Config) {
+  let returnVal = {};
+
+  let isBranchConfigured = has(config, 'branch') && !config.branch.includes('optional-branch-name');
+  let isPullRequestConfigured = has(config, 'pullRequest') && !config.pullRequest.includes('optional-pull-request-id');
+
+  if ( isBranchConfigured || isPullRequestConfigured) {
+    if ( isBranchConfigured ) {
+      returnVal = { branch: config.branch };
```

Comment:
why not return directly?

Reply:
My apologies. I'm working in another project in C and we use only one return at the end of function.  Updated

Reply:
you don't have to apologise, you did a great job, my remarks are only improvements nothing else 🎉